sycl/doc/EnvironmentVariables.md

```diff
@@ -28,6 +28,7 @@ subject to change. Do not rely on these variables in production code.
 | SYCL_DEVICELIB_NO_FALLBACK | Any(\*) | Disable loading and linking of device library images |
 | SYCL_PI_LEVEL_ZERO_MAX_COMMAND_LIST_CACHE | Positive integer | Maximum number of oneAPI Level Zero Command lists that can be allocated with no reuse before throwing an "out of resources" error. Default is 20000, threshold may be increased based on resource availabilty and workload demand. |
 | SYCL_PI_LEVEL_ZERO_DISABLE_USM_ALLOCATOR | Any(\*) | Disable USM allocator in Level Zero plugin (each memory request will go directly to Level Zero runtime) |
+| SYCL_PI_LEVEL_ZERO_USM_ALLOCATOR | MaxPoolableSize,Capacity,MaxPoolSize | Maximum allocation size in MB that may be pooled, number of pooled allocations in each size bucket, maximum size of pool in MB. Values specified as positive integers. Defaults are 1, 4, 256. |
```

Comment:
> number of pooled allocations in each size bucket

That one is not clear, users don't know about buckets, can it be clarified or eliminated?

Reply:
For some uniformity with OpenMP controls we want to retain this ability to control the pool. I have clarified the Capacity setting.